blast_plugins/radioplayer.py

```diff
@@ -80,7 +80,7 @@ def poke_track(self, now_playing: Optional[NowPlaying]):
         r = requests.post(
           url,
           auth = auth,
-          data = data
+          params = data
```

Comment:
Why this change?

Reply:
It seems params uses URL parameters, actually matching the cURL example above in the code. It seems the new API doesn't accept the data as POST data, only params, so was giving a 500 error.